test/extensions/INTEL/SPV_INTEL_fpga_reg/IntelFPGAReg.ll

```diff
@@ -172,21 +172,21 @@ entry:
   call void @llvm.lifetime.start.p0i8(i64 4, i8* %2) #4
   %3 = load i32, i32* %a, align 4, !tbaa !9
   ; CHECK-SPIRV: FPGARegINTEL [[TYPE_INT32]] {{[0-9]+}} {{[0-9]+}}{{ *$}}
-  ; CHECK-LLVM-DAG: %{{[0-9]+}} = call i32 @llvm.annotation.i32.p0i8(i32 {{[%a-z0-9]+}}, i8* getelementptr inbounds ([25 x i8], [25 x i8]* @[[STR]]
-  %4 = call i32 @llvm.annotation.i32(i32 %3, i8* getelementptr inbounds ([25 x i8], [25 x i8]* @.str, i32 0, i32 0), i8* getelementptr inbounds ([9 x i8], [9 x i8]* @.str.1, i32 0, i32 0), i32 35)
+  ; CHECK-LLVM-DAG: %{{[0-9]+}} = call i32 @llvm.annotation.i32.p0(i32 {{[%a-z0-9]+}}, ptr @[[STR]]
+  %4 = call i32 @llvm.annotation.i32(i32 %3, i8* bitcast ([25 x i8]* @.str to i8*), i8* bitcast ([9 x i8]* @.str.1 to i8*), i32 35)
```

Comment:
actually, should I convert the test to opaque pointers completely?

Reply:
I just opened #2084 for converting all the tests I can to use opaque pointers in reverse translation. (The main tests I don't have in this mode are the joint matrix tests, because of the nature of their test checks).